sample/android/gradle.properties

```diff
@@ -26,3 +26,7 @@ android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
 FLIPPER_VERSION=0.37.0
+
+sentry.collectModulesScript=../../../dist/js/tools/collectModules.js
+sentry.modulesPaths=node_modules,../..
```

Comment:
Is this mandatory for a final App adding the SDK?

Reply:
No, this is just override that is needed for our sample, but there are defaults that should be good for basically everyone